### DISCUSSION TITLE
增加 compose第三方 社区应用

### DISCUSSION THREAD

**Discussion by docker-master** [istoreos/istoreos#1777] [Ideas]
![image](https://github.com/user-attachments/assets/376b257b-6d1f-4af1-9e01-3db066337320)
    发现 路由狗 的应用商店里面有       “社区应用”里的 “NAS生态”
    介于制作ipk有点麻烦，可以开放一个 可以添加 compose 第三方“源”  的 应用商店
    就可以 把类似于 “矿神源 ”  的第三方 compose 应用添加进来安装，像casaOS一样的有5,6个“源仓库”
这样就可以有###  **无限的应用**

**Comment by zzh4141** [comment]
这个主意很不错,也希望官方可以添加这个功能

**Comment by jjm2473** [comment]
有源地址吗，我看看是啥格式的。如果只有docker compose文件，还是一样要写插件提供WEB配置界面。

**Reply by docker-master** [reply]
https://awesome.casaos.io/content/3rd-party-app-stores/list.html
![image](https://github.com/user-attachments/assets/3d95c07e-15b2-4635-bf99-c7b5fec3fb13)
这里有8个仓库，都是GitHub的，点进去是这样
![image](https://github.com/user-attachments/assets/62745ab2-fc7f-4870-ad01-7d0475122e52)
![image](https://github.com/user-attachments/assets/69f2847b-e8ad-4e1f-a20d-7427a20b9ef1)
再多的我也不懂了，出发你下载他的系统看看
